src/pages/components/menu-buttons/code.mdx

```diff
@@ -158,6 +162,19 @@ each framework below.
 </Column>
 </Row>
 
+### Feature flags
+
+A
+[feature flag](https://carbondesignsystem.com/components/overview/feature-flag/)
+has been added to the overflow menu variant of menu buttons to improve
+accessibility and changes parts of its functionality, not its visual appearance.
+For more code-specific feature flag information, refer to the
+[@carbon/react](https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-overflowmenu--overview)
+framework.
+
+The `enable-v12-overflowmenu` flag improves accessibility by more closely
+following the APG pattern for Menu and supports all the new Menu primitives.
```

Comment:
Just making sure if this Menu, all Menu buttons, or the overflow menu?

Reply:
Good point! Since it's only for the subcomponent of the Menu button (i.e., Overflow menu), I’ll check with @laurenmrice (once she's back from her vacation) to confirm if any updates are needed.

Reply:
I think it clarifies [here ](https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags-overflowmenu--overview) and [subcomponents of Menu](https://react.carbondesignsystem.com/?path=/story/components-menu--playground) (or context menu)

![image](https://github.com/user-attachments/assets/6303d90f-3d5f-47b8-a002-5db2b6da7c86)

Reply:
Yeah right, I checked that. Lauren wrote that,  I just want to double-check with her before making the change. Thanks for sharing that!